src/plugins/console/api_server/es_5_0/aggregations.js

```diff
@@ -0,0 +1,436 @@
+var simple_metric = {
+  __template: {field: ""},
+  field: "{field}",
+  missing: 0,
+  script: {
+    // populated by a global rule
+  }
+}, field_metric = {
```

Comment:
As i mentioned above, this is cool to be pushed to something else, this is a big PR already.